lsp-pwsh.el

```diff
@@ -256,25 +260,24 @@ Must not nil.")
 (defun lsp-pwsh--extra-init-params ()
   "Return form describing parameters for language server.")
 
-(defun lsp-pwsh--apply-code-action-edits (action)
+(lsp-defun lsp-pwsh--apply-code-action-edits ((&Command :command :arguments?))
   "Handle ACTION for PowerShell.ApplyCodeActionEdits."
-  (-if-let* ((command (gethash "command" action))
-             ((&hash "StartLineNumber" "EndLineNumber"
-                     "StartColumnNumber" "EndColumnNumber" "Text")
-              (lsp-seq-first (gethash "arguments" action)))
+  (-if-let* (((&pwsh:ScriptRegion :start-line-number :end-line-number
+                                  :start-column-number :end-column-number :text)
+              (lsp-seq-first arguments?))
              (edits `[,(ht ("range" (ht ("start"
-                                         (ht ("line" (- StartLineNumber 1))
-                                             ("character" (- StartColumnNumber 1))))
+                                         (ht ("line" (- start-line-number 1))
+                                             ("character" (- start-column-number 1))))
                                         ("end"
-                                         (ht ("line" (- EndLineNumber 1))
-                                             ("character" (- EndColumnNumber 1))))))
-                           ("newText" Text))]))
+                                         (ht ("line" (- end-line-number 1))
+                                             ("character" (- end-column-number 1))))))
+                           ("newText" text))]))
```

Comment:
These should be changed to `lsp-make-text-edit` and `lsp-make-range`

Reply:
I made this change and really forgot to push :sweat_smile:
Thanks for noticing it